base_konnector.js

```diff
@@ -41,6 +41,17 @@ module.exports = {
           process.exit(0)
         })
         .then(account => {
+          // get the folder path from the folder id and put it in cozyFields.folderPath
+          return new Promise((resolve, reject) => {
+            cozy.files.statById(cozyFields.folderPath, false)
```

Comment:
If I understand well, `folderPath` is actually an id ? Is it expensive to rename it ?